src/components/policy/test/sql_pt_representation_test.cc

```diff
@@ -68,11 +76,18 @@ class SQLPTRepresentationTest : public SQLPTRepresentation,
   static DBMS* dbms;
   static SQLPTRepresentation* reps;
   static const std::string kDatabaseName;
+  //Gtest can show message that this object doesn't destroyed
+  static utils::SharedPtr<policy_handler_test::MockPolicySettings> policy_settings_;
```

Comment:
@AByzhynar It is a way to avoid google error with not deleted static mock object. It cannot be changed to just object, because gtest will still show error with not deleted object. We cannot use `AllowLeak` because it still create problem in finally destruction in test_main.
It should be a static, because we should open real database only ones, if we open it in every test, the speed will be decreased. That's why we should use here static object, and it should be created as shared pointer